index.js

```diff
@@ -26,6 +27,8 @@ BroccoliMergeTrees.prototype.build = function() {
   if (this.mergeTrees == null) {
     // Defer instantiation until the first build because we only
     // have this.inputPaths and this.outputPath once we build.
+    this.outputPath = this.options.destDir ? this.outputPath + '/' + this.options.destDir : this.outputPath;
+    fs.ensureDirSync(this.outputPath);
```

Comment:
Can we avoid the extra IO operation if not needed? (it wasn't not always needed before)

Reply:
Fixed this.